app/javascript/components/pages/YelpSearch.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import { Button, Form, FormGroup, Input, Label } from "reactstrap";
+import { Redirect } from "react-router-dom";
+import YelpBusinesses from "./YelpBusinesses";
+
+export default class YelpSearch extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: {
+        item: "",
+        location: "",
+      },
+      submitted: false,
+    };
+  }
+
+  handleChange = (e) => {
+    let { form } = this.state;
+    form[e.target.name] = e.target.value;
+    this.setState({ form: form });
+  };
+
+  handleSubmit = () => {
+    this.setState({ submitted: true });
+  };
+  render() {
+    const { yelpApi, businesses } = this.props;
+    return (
+      <div>
+        <h1>Search For Sights Near You</h1>
+        <Form>
+          <FormGroup>
+            <Label for="item">
+              What are you in the mood for?{" "}
+              <span>e.i 'museum', 'food', 'park'</span>
+            </Label>
+            <Input
+              type="text"
+              name="item"
+              onChange={this.handleChange}
+              value={this.state.form.item}
+            />
+          </FormGroup>
+          <br />
+          <FormGroup>
+            <Label for="location">
+              Location:{" "}
```

Comment:
`{" "}` resolves to empty string so I think it can be removed

Reply:
with prettier it keeps putting it back because their was a space there